functests/5_latency_testing/latency_testing.go

```diff
@@ -160,6 +164,41 @@ var _ = Describe("Run tests of latency measurement tools with different values o
 			table.Entry("[test_id:42856] Hwlatdetect - Verify that the latency tool test should print an expected error message when passing invalid environment variables values", getNegativeTests(hwlatdetect), negativeTesting),
 		)
 	})
+
+	Context("Verify tests output with updated performance profile", func() {
+		It("[test_id:52388] should skip oslat test when isolated CPUs amount is less than 2", func() {
```

Comment:
is this a backport of a existing test?

Reply:
no this is a new test, we need to have it in 4.11 as well. currently 4.11 latency patches are waiting for the branching to avoid creating BZ, so testing this on 4.10 first once approved will create a PR for 4.11